selfdrive/ui/ui.hpp

```diff
@@ -37,36 +37,20 @@
 
 #define UI_BUF_COUNT 4
 
-// TODO: Detect dynamically
-#ifdef QCOM2
-const int vwp_w = 2160;
-#else
-const int vwp_w = 1920;
-#endif
+typedef struct Rect {
+  int x, y, w, h;
+  int centerX() const {return x+w/2;}
+  int right() const {return x+w;}
+  int bottom() const {return y+h;}
+  bool ptInRect(int px, int py) const{ return px>=x&&px<(x+w)&&py>=y&&py<(y+h); }
```

Comment:
add some whitespace and/or use another line

Reply:
done